PhysicsTools/Utilities/src/LumiReWeighting.cc

```diff
@@ -157,21 +157,7 @@ double LumiReWeighting::weight( float npv ) {
 
 double LumiReWeighting::weight( const edm::EventBase &e ) {
 
-  // find provenance of event objects, just to check at the job beginning if there might be an issue  
-
-  if(FirstWarning_) {
-
-    const edm::ProcessHistory& PHist = e.processHistory();
-    edm::ProcessHistory::const_iterator PHist_iter = PHist.begin();
-
-    for(; PHist_iter<PHist.end() ;++PHist_iter) {
-      edm::ProcessConfiguration PConf = *(PHist_iter);
-
-    }
-    //    SetFirstFalse();
-    FirstWarning_ = false;
-  }
-
+  FirstWarning_ = false; // in the previous statement, FirstWarning_ would've been turned to false if it were true, and would've remain false if not, false in both cases
```

Comment:
@mrodozov - this variable is set, but not used.

Reply:
@ianna - it's a memvar https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/Utilities/interface/LumiReWeighting.h#L257 that signifies some state